python/lsst/ts/m2gui/main_window.py

```diff
@@ -95,6 +95,7 @@ def __init__(self, is_output_log_on_screen, log=None):
         # Set the default conditions
         self._layout_control.set_csc_commander(True)
         self.model.report_config()
```

Comment:
Would be better to add a function into model to setup? def setDefaults, which will handle all init calls?

Reply:
Fixed in the commit: [4d02847](https://github.com/lsst-ts/ts_m2gui/pull/7/commits/4d028476f7478675d575228b95db083bc1ae6e33). Thanks!